src/com/owncloud/android/ui/activity/FileActivity.java

```diff
@@ -346,7 +360,7 @@ protected void initDrawer(){
         // mDrawerItems.add(new NavigationDrawerItem(mDrawerTitles[0],
         // mDrawerContentDescriptions[0]));
         // All Files
-        mDrawerItems.add(new NavigationDrawerItem(mDrawerTitles[0], mDrawerContentDescriptions[0]));
+        mDrawerItems.add(new NavigationDrawerItem(mDrawerTitles[0], mDrawerContentDescriptions[0],R.drawable.ic_folder_open));
```

Comment:
Please, keep lines shorter than 120 characters. It's the maximum shown by Github in pull request.